tchannel/peer_heap.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from .container import heap
+from .container.heap import HeapOperation
+
+
+class PeerHeap(HeapOperation):
+    def __init__(self):
+        self.peers = []
```

Comment:
There won't be pop action all the time. The whole implementation of heap is depending on list. List has instance access in case we want to update score for some peers directly.